versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/common-issues.md

```diff
@@ -0,0 +1,11 @@
+---
+title: AWS Marketplace Common Issues
+---
+
+1. Uninstalling Rancher Prime may not cleanly remove all the resources that Rancher created. You are encouraged to use the [Rancher resource cleanup script](https://github.com/rancher/rancher-cleanup) to perform a more comprehensive cleanup if necessary. However, we recommend first preparing and migrating any other workloads off the cluster to destroy the cluster to complete the uninstallation since cleanup is not recoverable.
+
+   ```shell
+   helm uninstall -n cattle-rancher-csp-deployer-system rancher-cloud
+   ```
```

Comment:
The entirety of this first common issue entry is coupled with `# Uninstalling Rancher Prime PAYG Offering` from `install-rancher-prime.md`, maybe we can move it there?